policyengine-client/src/policyengine/general/navigationButton.jsx

```diff
@@ -5,18 +5,19 @@ import { CountryContext } from "../../countries";
 import { policyToURL } from "../tools/url";
 
 export default function SimulateButton(props) {
-    const country = useContext(CountryContext);
+	const country = useContext(CountryContext);
 	const url = policyToURL(`/${country.name}/${props.target}`, country.policy);
-	let button = <Button 
-			disabled={props.disabled} 
-			type={props.primary ? "primary" : null} 
-			onClick={props.onClick}>{props.text}
-		</Button>;
-	if(props.target) {
+	let button = <Button
+		disabled={props.disabled}
+		type={props.primary ? "primary" : null}
+		onClick={props.onClick}>{props.text}
+		block
```

Comment:
the formatting changes make it hard to see, but this is the only actual change I made @nikhilwoodruff